namelayer-spigot/src/main/java/vg/civcraft/mc/namelayer/database/GroupManagerDao.java

```diff
@@ -1004,21 +844,43 @@ public void run() {
 	}
 	
 	public void removeMember(UUID member, Group group){
-		//TODO Redo this
+		//TODO Redo remove member
 		try (Connection connection = db.getConnection();
-				PreparedStatement removeMember = connection.prepareStatement(this.removeMember)){
+				PreparedStatement removeMember = connection.prepareStatement(GroupManagerDao.removeMember)){
 			removeMember.setString(1, member.toString());
 			//removeMember.setString(2, group);
 			removeMember.executeUpdate();
 		} catch (SQLException e) {
 			logger.log(Level.WARNING, "Problem removing " + member + " from group " + group, e);
 		}
 	}
+
+	public void removeAllMembersAsync(final Group group){
```

Comment:
Some of these methods I've left in place w.r.t. potential future uses, not because they have immediate use. :shrug: I get we want clean code but API completeness is nice too.